tracing/src/test/java/com/palantir/tracing/TracersTest.java

```diff
@@ -404,10 +400,8 @@ public void testWrapFuture_failure() {
             ListenableFuture<String> traced = Tracers.wrapListenableFuture(
                     operationName, () -> Futures.immediateFailedFuture(new SafeRuntimeException("result")));
             assertThat(traced).isDone();
-            assertThat(observed).hasSize(2);
-            // Inner operation must complete first to avoid confusion
-            assertThat(observed.get(0)).extracting(Span::getOperation).isEqualTo(operationName + " initial");
-            assertThat(observed.get(1)).extracting(Span::getOperation).isEqualTo(operationName);
+            assertThat(observed).hasSize(1);
+            assertThat(observed.get(0)).extracting(Span::getOperation).isEqualTo(operationName);
```

Comment:
Everyone using the `Tracers.wrapListenableFuture` functionality will get cleaner tracing spans (no more "initial" span) when they upgrade.